lib/cloud.proxy.js

```diff
@@ -363,9 +363,41 @@ IoTKitCloud.prototype.disconnect = function () {
     me.proxy.disconnect();
 };
 
+IoTKitCloud.prototype.refreshSpbMetricList = function() {
```

Comment:
Will optimize refresh only to a newly registered device component and add SpB device birth message to that component as well